.github/workflows/ci_tests.yaml

```diff
@@ -15,30 +15,14 @@ defaults:
     shell: bash
 
 jobs:
-  gosec:
-    name: Run gosec scan
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Gatekeeper
-        uses: actions/checkout@v4
-      - name: Run Gosec Security Scanner
-        uses: securego/gosec@v2.15.0
-        with:
-          args: -no-fail -fmt sonarqube -out gosec.json -stdout -exclude-dir=.go -exclude-dir=test ./...
-      - name: Upload artifacts
-        uses: actions/upload-artifact@v3
-        with:
-          name: artifacts
-          path: gosec.json
```

Comment:
The `upload-artifacts` workflow changed such that uploading files to an artifact of the name name is no longer possible, so the GoSec scan has been moved to the unit tests, where coverage is also uploaded.